app/bundles/LeadBundle/Entity/LeadFieldRepository.php

```diff
@@ -29,7 +29,7 @@ class LeadFieldRepository extends CommonRepository
      *
      * @return array
      */
-    public function getAliases($exludingId, $publishedOnly = false, $includeEntityFields = true, $object = 'lead')
+    public function getAliases($exludingId, $publishedOnly = false, $includeEntityFields = true, $object = null)
```

Comment:
This is BC for me, I recommend let it 

`$object = 'lead'` 

and bellow use

`$repo->getAliases($field->getId(), false, true, null);`

Somebody can use in own environment and this change could brake compatibility

What do you think?

Reply:
@kuzmany are you still pending feedback on this issue, or is it OK for merging with two successful tests?

Reply:
It's public function. Then for me it's BC issue.
@escopecz Do you have opinion on that?

Reply:
The code was updated, so there is no issue with this. It can be merged as it has 2 approvals.